websites/N/Notespace/presence.ts

```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+	clientId: "1193651901221306378",
+});
+
+const enum Assets {
+	icon = "icon",
+}
+
+presence.on("UpdateData", async () => {
+	const currentPath = document.location.pathname,
+		presenceData: PresenceData = {
+			details: "Twoja wiedza jest tu",
+			largeImageKey: Assets.icon,
+			startTimestamp: new Date(),
+			buttons: [
+				{ label: "Odwiedź stronę", url: "https://notespace.edu.pl/" },
```

Comment:
[Buttons](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) that redirect to main pages are not allowed.

Reply:
very sorry for that